src/Umbraco.Core/EmbeddedResources/Lang/da.xml

```diff
@@ -366,16 +366,16 @@
     <key alias="noDocumentTypes" version="7.0">
       <![CDATA[Der kunne ikke findes nogen tilladte dokumenttyper. Du skal tillade disse i indstillinger under <strong>"dokumenttyper"</strong>.]]></key>
     <key alias="noDocumentTypesAtRoot">
-      <![CDATA[There are no document types available for creating content here. You must create these in <strong>Document Types</strong> within the <strong>Settings</strong> section.]]></key>
+      <![CDATA[Der er ingen dokumenttyper tilgængelige til at oprette indhold her. Du skal oprette disse under <strong>Dokumenttyper</strong> i <strong>Indstillinger</strong> sektionen.]]></key>
```

Comment:
Compound words (sammensatte ord) should be without the space in Danish. Optionally a hyphen may be used instead.

So I'd argue `<strong>Indstillinger</strong>-sektionen` would be more correct.

Reply:
👍